### PR TITLE
feature/ adding contact_names as a new field to whois's proto and cha…

### PR DESCRIPTION
**Pull request by PiranhaSa**
…nging the old contact_name field to be deprecated

### CODE REVIEW

File: src/ostorlab/agent/message/proto/v3/asset/domain_name/whois/whois.proto [hunk 1]
```diff
@@ -14,11 +14,12 @@ message Message {
...
   repeated string dnssec = 11;
-  optional string contact_name = 12;
+  optional string contact_name = 12 [deprecated=true];
   optional string org = 13;
   optional string address = 14;
   optional string city = 15;
   optional string state = 16;
   optional string zipcode = 17;
   optional string country = 18;
+  repeated string contact_names = 19 ;
```
Comment (benyissa) [inline src/ostorlab/agent/message/proto/v3/asset/domain_name/whois/whois.proto:24]:
```suggestion
  repeated string contact_names = 19;
```

File: src/ostorlab/agent/message/proto/v3/asset/domain_name/whois/whois_pb2.py [hunk 2]
```diff
@@ -13,13 +13,185 @@
 
 
 
-DESCRIPTOR = _descriptor_pool.Default().AddSerializedFile(b'\nCostorlab/agent/message/proto/v3/asset/domain_name/whois/whois.proto\x12\x37ostorlab.agent.message.proto.v3.asset.domain_name.whois\"\xd5\x02\n\x07Message\x12\x0c\n\x04name\x18\x01 \x01(\t\x12\x11\n\tregistrar\x18\x02 \x01(\t\x12\x14\n\x0cwhois_server\x18\x03 \x01(\t\x12\x14\n\x0creferral_url\x18\x04 \x01(\t\x12\x14\n\x0cupdated_date\x18\x05 \x03(\t\x12\x15\n\rcreation_date\x18\x06 \x03(\t\x12\x17\n\x0f\x65xpiration_date\x18\x07 \x03(\t\x12\x14\n\x0cname_servers\x18\x08 \x03(\t\x12\x0e\n\x06status\x18\t \x03(\t\x12\x0e\n\x06\x65mails\x18\n \x03(\t\x12\x0e\n\x06\x64nssec\x18\x0b \x03(\t\x12\x14\n\x0c\x63ontact_name\x18\x0c \x01(\t\x12\x0b\n\x03org\x18\r \x01(\t\x12\x0f\n\x07\x61\x64\x64ress\x18\x0e \x01(\t\x12\x0c\n\x04\x63ity\x18\x0f \x01(\t\x12\r\n\x05state\x18\x10 \x01(\t\x12\x0f\n\x07zipcode\x18\x11 \x01(\t\x12\x0f\n\x07\x63ountry\x18\x12 \x01(\t')
+DESCRIPTOR = _descriptor.FileDescriptor(
```
Comment (deadly-panda) [inline src/ostorlab/agent/message/proto/v3/asset/domain_name/whois/whois_pb2.py:16]: Which version of protoc did you use to compile ?

**Review by deadly-panda — CHANGES_REQUESTED** [review]

File: src/ostorlab/agent/message/proto/v3/asset/domain_name/whois/whois.proto (hunk 1 above)
Comment (3asm) [reply src/ostorlab/agent/message/proto/v3/asset/domain_name/whois/whois.proto:24]: +1

File: src/ostorlab/agent/message/proto/v3/asset/domain_name/whois/whois_pb2.py (hunk 2 above)
Comment (PiranhaSa) [reply src/ostorlab/agent/message/proto/v3/asset/domain_name/whois/whois_pb2.py]: libprotoc 25.0-rc2

**Review by ErebusZ — APPROVED** [review]

**Review by deadly-panda — APPROVED** [review]

**Comment by PiranhaSa** [comment]
Don't merge please

**Pull request closed without merging**